src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -184,8 +184,8 @@ export class CodeSuggestionWebviewProvider extends WebviewProvider implements IC
       'codeSuggestionWebviewScript.js',
     );
     const styleUri = this.getWebViewUri('media', 'views', 'snykCode', 'suggestion', 'suggestion.css');
+    const styleVSCodeUri = this.getWebViewUri('media', 'views', 'featureSelection', 'vscode.css');
```

Comment:
Let's move `views/featureSelection/vscode.css` to `views/common/vscode.css` since this is used across two different views now.

Reply:
Latest screenshot: 
![image](https://user-images.githubusercontent.com/20150761/141789553-3272b580-9f65-4cae-beca-72679cf19cc8.png)